warehouse/macaroons/services.py

```diff
@@ -54,17 +54,12 @@ def find_macaroon(self, macaroon_id):
         Returns a macaroon model from the DB by its identifier.
         Returns None if no macaroon has the given ID.
         """
-        try:
-            dm = (
-                self.db.query(Macaroon)
-                .options(joinedload("user"))
-                .filter(Macaroon.id == uuid.UUID(macaroon_id))
-                .one()
-            )
-        except NoResultFound:
-            return None
-
-        return dm
+        return (
+            self.db.query(Macaroon)
+            .options(joinedload("user"))
+            .options(joinedload("project"))
+            .get(macaroon_id)
```

Comment:
This is deprecated, see https://docs.sqlalchemy.org/en/14/changelog/migration_20.html#migration-orm-usage

Reply:
af976cdd